src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,5 +1,7 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
```

Comment:
```suggestion
public class AuthenticationService {
    private final UserService userService = new UserService();
```

Reply:
I recceive the message: "Modifier 'private' is not allowed here"

Reply:
There is no any error messages
<img width="816" alt="Screenshot 2024-12-10 at 17 51 17" src="https://github.com/user-attachments/assets/4a471f40-f60b-415c-9efe-ed3f11dc353c">